core/src/test/scala/org/apache/spark/util/UtilsSuite.scala

```diff
@@ -17,8 +17,7 @@
 
 package org.apache.spark.util
 
-import java.io.{ByteArrayInputStream, ByteArrayOutputStream, DataOutput, DataOutputStream, File,
-  FileOutputStream, InputStream, PrintStream, SequenceInputStream}
+import java.io._
```

Comment:
I would revert this, but it's not a big deal.

Reply:
+1 for @srowen 's comment.

Reply:
done